wormhole/wormhole.go

```diff
@@ -90,6 +97,14 @@ func (c *Client) wordCount() int {
 }
 
 func (c *Client) relayURL() internal.SimpleURL {
+	if c.TransitRelayAddress != "" {
+		relayUrl, err := internal.NewSimpleURL(c.TransitRelayAddress)
+		if err != nil {
+			return relayUrl
+		} else {
+			errors.New("Malformed Transit Relay Address")
```

Comment:
Thanks for the catch. I guess the compiler didn't catch that because of the missing `return` statement.